Block/PageType.php

```diff
@@ -83,14 +57,19 @@ public function __construct(
      */
     private function getPageTypeName()
     {
-        return $this->getData('page_type') ? $this->getData('page_type') : self::DEFAULT_TYPE;
+        return $this->getData('page_type') ?: self::DEFAULT_TYPE;
     }
 
-    public function getAbstractObject()
+    /**
+     * Returns the HTML to render PageTypes
+     *
+     * @return string
+     * @suppress PhanUndeclaredClassMethod
+     */
+    public function toHtml()
```

Comment:
Same thing than with the category. No need to override the `toHtml`. Keep the `getAbstractObject` method and return the object

Reply:
Reverted